app/services/app/app.ts

```diff
@@ -103,9 +103,20 @@ export class AppService extends StatefulService<IAppState> {
       });
     }
 
-    // We want to start this as early as possible so that any
-    // exceptions raised while loading the configuration are
-    // associated with the user in sentry.
+    // perform several concurrent http requests
+    await Promise.all([
+      // We want to start this as early as possible so that any
+      // exceptions raised while loading the configuration are
+      // associated with the user in sentry.
+      this.userService.validateLogin(),
+
+      // this config should be downloaded before any game-capture source has been added to the scene
+      this.downloadAutoGameCaptureConfig(),
+    ]).catch(e => {
+      // probably the internet is disconnected
+    });
+
+    this.downloadAutoGameCaptureConfig();
```

Comment:
I don't think we want to be doing this twice.

Reply:
Good catch. Updated